src/explore-education-statistics-frontend/src/modules/download/DownloadIndexPage.tsx

```diff
@@ -57,7 +58,16 @@ const DownloadIndexPage: NextPage<Props> = ({ themes = [] }) => {
       </div>
 
       {themes.length > 0 ? (
-        <Accordion id="downloads">
+        <Accordion
+          id="downloads"
+          onToggle={accordionSection => {
```

Comment:
Let's rename `onToggle` to `onSectionOpen`. `onToggle` is actually misleading as it doesn't actually get called when a section is closed (the prop name would suggest otherwise though).